exercises/space-age/description.md

```diff
@@ -4,7 +4,7 @@ Given an age in seconds, calculate how old someone would be on:
 
 - Mercury: orbital period 0.2408467 Earth years
 - Venus: orbital period 0.61519726 Earth years
-- Earth: orbital period 1.0 Earth years, 365.25 Earth days, or 31557600 seconds
+- Earth: orbital period 1.0 Earth years, 365.2425 Earth days, or 31557600 seconds
```

Comment:
if 365.25 is changed, 31557600 must change as well, because 365.2425 * 24 * 3600 does not equal 31557600

this comment is not to be taken as support for nor opposition against changing 365.25